### ISSUE TITLE
show market price in INR

### ISSUE THREAD

**Issue by pranavk** [cryptonium-app/cryptonium-app.github.io#2]
If I change the currency to INR then it shows me a direct conversion of USD to INR. But I would never be able to buy in India at that price. So it should rather show the rate according to the market rate in the country rather than just using the direct USD <-> INR conversion?

**Comment by akshendra** [comment]
We are using coinmarketcap and cryptocompare apis. On changing the currency, the data is fetch again for the new currency. I am not sure how they are doing the conversion.

**Comment by sagarkarira** [comment]
@pranavk Yeah this is a quite a challenge. You need to fetch data from local exchanges as prices vary from countries to countries. Coinmarketcap does not provide that filter in their APIs so you have to do a real time listing of all the exchange prices. 

We hope to do this in future. 😄
